Makefile

```diff
@@ -176,14 +178,27 @@ else
 KUSTOMIZE=$(shell which kustomize)
 endif
 
+OPERATOR_SDK = $(BINDIR)/operator-sdk
+operator-sdk:
+ifeq (,$(wildcard $(OPERATOR_SDK)))
+	@{ \
+	set -e ;\
+	mkdir -p $(dir $(OPERATOR_SDK)) ;\
+	curl -sSLo $(OPERATOR_SDK) https://github.com/operator-framework/operator-sdk/releases/download/$(OPERATOR_SDK_VERSION)/operator-sdk_${OS}_${ARCH};\
+	chmod +x $(OPERATOR_SDK) ;\
```

Comment:
Don't we need to `chmod` the actual `OPERATOR_SDK` binary? doesn't `OPERATOR_SDK` point to the directory?

Reply:
- it's path to the binary not to the directory

Reply:
ok